src/Storage/Models/InstanceQueryParameters.cs

```diff
@@ -25,6 +25,8 @@ public class InstanceQueryParameters
         private const string _instanceOwnerPartyIdParameterName = "instanceOwner.partyId";
         private const string _instanceOwnerPartyIdsParameterName = "instanceOwner.partyIds";
         private const string _lastChangedParameterName = "lastChanged";
+        private const string _mainVersionExcludeParameterName = "mainVersionExclude";
+        private const string _mainVersionIncludeParameterName = "mainVersionInclude";
```

Comment:
Request headers usually don't use lowerCamelCase. See the _instanceOwnerIdentifierHeaderName value.